app/components/articleSearch/components/requestPaperDialog/index.tsx

```diff
@@ -0,0 +1,167 @@
+import * as React from "react";
+import * as Cookies from "js-cookie";
+import * as classNames from "classnames";
+import { connect, Dispatch } from "react-redux";
+import { Formik, FormikErrors, Form, Field } from "formik";
+import { RouteComponentProps, withRouter } from "react-router-dom";
+import FeedbackManager from "@pluto_network/scinapse-feedback";
+import Dialog from "@material-ui/core/Dialog";
+import { CurrentUser } from "../../../../model/currentUser";
+import { withStyles } from "../../../../helpers/withStylesHelper";
+import { AppState } from "../../../../reducers";
+import validateEmail from "../../../../helpers/validateEmail";
+import ScinapseFormikInput from "../../../common/scinapseInput/scinapseFormikInput";
+import ReduxAutoSizeTextarea from "../../../common/autoSizeTextarea/reduxAutoSizeTextarea";
+import { ACTION_TYPES } from "../../../../actions/actionTypes";
+import ActionTicketManager from "../../../../helpers/actionTicketManager";
+import Icon from "../../../../icons";
+declare var ga: any;
+const styles = require("./requestPaperDialog.scss");
+
+const LAST_SUCCEEDED_EMAIL_KEY = "l_s_e_k";
```

Comment:
why you re-declare this constant again? it's in `fullTextDialog.tsx`. if you want to use this key in other places, just pull out the constant from `fullTextDialog.tsx` and make some `constant.tsx` in a proper position. then import & use it.